env.local

```diff
@@ -0,0 +1 @@
+REACT_APP_GITHUB_PERSONAL_ACCESS_TOKEN=245c634a5f72c5a1ed960c99bec2ff83b33da6b4
```

Comment:
It is best practice not to commit personal access tokens or api keys to GitHub. Exposing these keys make your application less secure.